### DISCUSSION TITLE
HM ES24TX v2.0

### DISCUSSION THREAD

**Discussion by mariusdp** [ExpressLRS/ExpressLRS#1079] [Q&A] [answered]
Hi,

Hope i am in the right place - if not sorry and please direct me... 10x
I got a HM ES24TX v2.0 (has a led plug and a fan plug as well) and I can not get the led (WS2812) to work. Tested on home made DIY ESP32_SX... and work fine. Is there a way to make it work? Traced the signal pin for the WS and it is connected to MTDO (GPIO15 - physical pin 21). Please if anyone knows how to get it working let me (and others) know. Thank you.

Regards and awesome work :)

**Comment by mariusdp** [comment]
Anyone? I would really use some help/suggestions.

**Reply by avsaase** [reply]
It's generally best to ask support questions on Discord: https://discord.gg/dS6ReFY.

**Reply by mariusdp** [reply]
10x. I'll give it a go later.

**Comment by mariusdp** [comment, marked as answer]
As suggested on discord (pointed there by avsaase) WS led and fan ctrl works only on 2.0.0-RC1. Great job all.

**Comment by Martijn0001** [comment]
Thanks for your update. I too am trying to get the LED to function. I have seen it work in my current installation, which runs on 1.2 but not reliably. Lets hope 2.0 does the trick. 
For others who might want to try, the connector has GROUND at the bottom. +5v in the middle, SIGNAL at the top.

**Reply by mariusdp** [reply]
Made one with ws led, fan and oled (no buzzer cause i do not see the point) - all works fine on 2.0.0rc3 (need to change a little for the i2c oled to work). The only thing that I think is missing is a button support for diy targets - hopefully will be added soon. :P
